crates/fluvio-cli-common/src/version_cmd.rs

```diff
@@ -0,0 +1,86 @@
+use anyhow::Result;
+use clap::Args;
+use current_platform::CURRENT_PLATFORM;
+use crate::FLUVIO_RELEASE_CHANNEL;
+use sha2::{Digest, Sha256};
+
+const VERSION: &str = include_str!("../../../VERSION");
+
+/// Display version information
+#[derive(Debug, Args)]
+pub struct BasicVersionCmd;
+
+impl BasicVersionCmd {
+    /// Display basic information about the current fluvio installation
+    ///
+    /// The following information is displayed:
+    /// - Release channel, if available;
+    /// - CLI version;
+    /// - Platform arch;
+    /// - CLI SHA256, if available;
+    /// - channel frontend SHA256, if available;
+    /// - Git hash, if available;
+    /// - OS details, if available;
+    pub fn process(self) -> Result<()> {
+        if let Ok(channel) = std::env::var(FLUVIO_RELEASE_CHANNEL) {
+            println!("Release Channel: {channel}");
+        }
+
+        println!("Fluvio CLI: {}", VERSION.trim());
+        println!("Fluvio CLI Arch: {CURRENT_PLATFORM}");
+
+        if let Some(sha) = self.format_cli_sha() {
+            println!("Fluvio CLI SHA256: {}", sha);
+        }
+
+        if let Some(sha) = self.format_frontend_sha() {
+            println!("Fluvio channel frontend SHA256: {}", sha);
+        }
```

Comment:
I don't know if we want to do it now, but when `fvm` is introduced, but this will need to be removed soon. Your call for how to deal with it

The main problem right now is we're likely to always print this line even though its context is only valid for `fluvio` and that can be a little confusing.

Reply:
removed